### ISSUE TITLE
Ship not giving up frame of reference after autopilot flight

### ISSUE THREAD

**Issue by SeanN** [pioneerspacesim/pioneer#34]
Pioneer dev build 771cbcc win 32
Windows vista 32
Nvidia 9600m

After autopiloting to a planet using the "fly to vicinity of..." or "enter high orbit around..." command, the frame of reference is not always the planet/moon I set as my destination.  An example:
1. Take off from Earth and autopilot to "vicinity of mars"  (the frame of reference eventually changes from Earth to Mars as you get closer and stop)
2. Select Phobos as a navigation target and use the autopilot to fly to the vicinity of Phobos (the ship stops a distance away from Phobos, but the frame of reference is still Mars-even though I am closer to Phobos)  If you activate the set speed control, the ship will begin accelerating relative to Mars.  Entering high orbit around Phobos also doesn't change the frame of reference.  Not until I enter a low or medium orbit does the frame of reference finally change to Phobos.

**Comment by Web-eWorks** [comment]
Should be fixed by #3957. Please file a new issue if you notice this behavior occuring in latest master.
